2271 - Fix button inside the popover (#2467)
infor-design/enterprise: docs/CHANGELOG.md

- `[Modal]` Fixed an issue where the modal component would disappear if its content had a checkbox in it in RTL. ([#332](https://github.com/infor-design/enterprise-ng/issues/332))
- `[Personalization]` Updated some of the colors to more readable in contrast mode. ([#2097](https://github.com/infor-design/enterprise/issues/2097))
- `[Popdown]` Fixed usability issue where the Popdown could close prematurely when attempting to use inner components, such as Dropdowns. ([#2092](https://github.com/infor-design/enterprise/issues/2092))

> - `[Popover]` Fixed an issue where x-button was not aligning. ([#1576](https://github.com/infor-design/enterprise/issues/1576))

Reviewer: ```suggestion
- `[Popover]` Fixed an issue where close button was not aligning. ([#1576](https://github.com/infor-design/enterprise/issues/1576))
```

- `[Modal]` Fixed an issue where the modal component would disappear if its content had a checkbox in it in RTL. ([#332](https://github.com/infor-design/enterprise-ng/issues/332))
- `[Personalization]` Updated some of the colors to more readable in contrast mode. ([#2097](https://github.com/infor-design/enterprise/issues/2097))
- `[Popdown]` Fixed usability issue where the Popdown could close prematurely when attempting to use inner components, such as Dropdowns. ([#2092](https://github.com/infor-design/enterprise/issues/2092))

> - `[Popover]` Fixed an issue where close button was not aligning. ([#1576](https://github.com/infor-design/enterprise/issues/1576))

Reviewer: ```suggestion
- `[Popover]` Fixed an issue where close button was not aligned correctly. ([#1576](https://github.com/infor-design/enterprise/issues/1576))
```

> - `[Popover]` Fixed an issue where close button was not aligned correctly. ([#1576](https://github.com/infor-design/enterprise/issues/1576))

Reviewer 2: ```suggestion
- `[Popover]` Correctly align the close button in a popover at smaller screen sizes. ([#1576](https://github.com/infor-design/enterprise/issues/1576))
```

> - `[Popover]` Fixed an issue where inside button was overflowing from container on mobile screen size. ([#2271](https://github.com/infor-design/enterprise/issues/2271))

Reviewer: ```suggestion
- `[Popover]` Fixed an issue where buttons inside the popover would overflow this popover container on smaller screen size. ([#2271](https://github.com/infor-design/enterprise/issues/2271))
```

> - `[Popover]` Fixed an issue where buttons inside the popover would overflow this popover container on smaller screen size. ([#2271](https://github.com/infor-design/enterprise/issues/2271))

Reviewer 2: ```suggestion
- `[Popover]` Fixed an issue where buttons inside the popover would overflow at smaller screen sizes. ([#2271](https://github.com/infor-design/enterprise/issues/2271))
```
